src/main/java/io/cdap/plugin/cdc/source/sqlserver/CTSQLServerConfig.java

```diff
@@ -58,42 +53,89 @@
   @Macro
   private final String username;
 
+  @Name(DBNAME)
+  @Description("Database Name")
+  public String dbName;
+
   @Name(PASSWORD)
   @Description("Password to use to connect to the specified database. Required for databases that " +
     "need authentication. Optional for databases that do not require authentication.")
   @Nullable
   @Macro
   private final String password;
 
+  @Name(SQN)
+  @Description("Need the CDC Sequence Number in the output schema")
+  @Nullable
+  @Macro
+  private final Boolean sqn;
+
+
+  @Name(CDCNUMBER)
+  @Description("CDC Sequence Number to start ingesting from")
+  @Nullable
+  @Macro
+  private final int cdcnumber;
+
+
+  @Name(TABLENAME)
+  @Description("Table name for CT Information")
+  @Nullable
+  @Macro
+  private final String tableName;
+
+  @Name(MAXBATCHSIZE)
+  @Description("Size of a record batch per pipeline batch interval")
+  @Nullable
+  @Macro
+  private final int maxBatchSize;
+
+  @Name(DRIVERCLASSNAME)
+  @Description("Driver class name")
+  @Nullable
+  @Macro
+  private final String driverClassName;
```

Comment:
why is this being added?

Apparently this plugin is packaging the jdbc driver instead of loading a jdbc plugin, which is bad behavior, but it means the classname shouldn't be configurable.

Reply:
I believe this class name is used to load the sql server driver, we should be able to rewrite the plugin so it loads the jdbc plugin